pages/search/[[term]].vue

```diff
@@ -75,6 +76,10 @@ onMounted(() => {
   isMounted.value = true;
   const main = ref<HTMLElement | null>(document.querySelector("main"));
 
+  if (searchbox.value) {
```

Comment:
This might be better to move out to a watcher instead of in `onMounted`, as the input might not exist yet here:

```ts
watch(searchbox, (box) => {
  if (box) {
    box.focus();
  }
});
```